1st draft icebreaker post (#319)
ropensci/roweb2: content/blog/2018-11-01-icebreaker.md

*Timed 3-minute chat among people standing near each other*

Please introduce yourself and tell us why you have chosen to stand where you are standing...

Does anyone wish to move based on what you’ve heard?

> Question 2: When I need to do something I’ve never done before, I a) consult an outside authority e.g. read the documentation or talk to an expert and try to understand the big picture OR b) I try to find a working example and modify it for my purposes

*Continue with brief introductions to the entire group for 10 to 12 minutes for each question*

> Question 3: I know where I fit as a member of the R community (agree vs disagree)

*Near end of the exercise, ask anyone who did not speak up to introduce themselves to the group.*

## Reflection

> Really work on the questions - they're the key to shifting it from being an ok ice breaker to a really provocative and useful workshop starter. Think about the questions that underpin the workshop, what are the issues people are worried about but unsure how to raise? Can you frame a slightly humorous question that makes it ok to talk about? - Cameron Neylon

> The “I know where I fit as a member of the R community” question was ideal coming at the end of the ice breaker because people had become comfortable with the flow of discussion and were willing to be a bit more vulnerable and share slightly more personal opinions. Someone noted that despite having equal representation of women and men in the group there was a much larger proportion of women who indicated “I don't know where I fit in the R community”. One acknowledged expert had themselves standing at the “I don't know where I fit in the R community” end, and they indicated that this was because they suffered from imposter syndrome. This gave me an opportunity to explain [imposter syndrome](https://en.wikipedia.org/wiki/Impostor_syndrome) and subsequently, when we polled the entire group, two thirds said they identified with it!

Reviewer: ```suggestion
The “I know where I fit as a member of the R community” question was ideal coming at the end of the ice breaker because people had become comfortable with the flow of discussion and were willing to be a bit more vulnerable and share slightly more personal opinions. Someone noted that there was a much larger proportion of women in the group who indicated “I don't know where I fit in the R community” than in the other group. One acknowledged expert had themselves standing at the “I don't know where I fit in the R community” end, and they indicated that this was because they suffered from imposter syndrome. This gave me an opportunity to explain [imposter syndrome](https://en.wikipedia.org/wiki/Impostor_syndrome) and subsequently, when we polled the entire group, two thirds said they identified with it!
```

Reviewer: The current phrasing sounds a bit binary. You can simply say the proportion of women was higher in one group than the other, after all this is the message no matter what the proportion of women is to begin with.

Some people who participated in this ice breaker at our unconferences have adapted it for their own events.

- Julia Stewart-Lowndes at xxx
- [Reka Solymosi](https://twitter.com/r_solymosi) uses it with 20 to 30 people at welcome week activities for graduate students in Criminology and Social Statistics at the University of Manchester. The questions she uses are: 1) I have a previous degree in/ have worked in a career related to criminology; 2) I know exactly what I want to gain out of this masters programme.
- [Miles McBain](https://twitter.com/milesmcbain) used it with 35 people at the 2018 [R Foundation](https://www.r-project.org/foundation/) Summit
- [Sam Albers](https://twitter.com/big_bad_sam) used it with 30 people at an ____ event/workshop for the ____ (e.g. R users in the British Columbia government dept xyz)
- [Nick Tierney](https://twitter.com/nj_tierney) used it with 45 people at [ozunconf17](https://ropensci.org/blog/2017/10/31/ozunconf2017/)

## Tips
1. Work hard to design questions that relate to your community and event. Start with a frivolous question and then dig deep.
1. Write a script for yourself; it forces you to think through the excercise.

> 1. Explain how the exercise works and find two volunteers to demonstrate

Reviewer: ```suggestion
1. Explain how the exercise works and find two volunteers to demonstrate.
```

Reviewer: my first time using the suggestion mode, sooo nice.
